plugins/moodbar/__init__.py

```diff
@@ -543,20 +608,41 @@ def enable(exaile):
     else:
         _enable(None, exaile, None)
 
+
 def _enable(eventname, exaile, nothing):
-    global ExaileModbar
-    ExaileModbar.readMod('')
-    ExaileModbar.setupUi()
-    event.add_callback(ExaileModbar.play_start, 'playback_track_start', player.PLAYER)
-    event.add_callback(ExaileModbar.play_end, 'playback_player_end', player.PLAYER)
+    _enable_main_moodbar(exaile)
+
+    event.add_callback(_enable_preview_moodbar, 'preview_device_enabled')
+    event.add_callback(_disable_preview_moodbar, 'preview_device_disabling')
+
+    try:
+        import previewdevice
```

Comment:
I don't like this import. I think there's a way to get the plugin object differently... well, I guess not.

I think the better option would be to add a call to the PluginManager like `get_enabled_plugin()`. All currently enabled plugins are in the enabled_plugins dictionary. The objects in there are the plugin modules.